edunext_openedx_extensions/edunext/management/commands/import_xml_db.py

```diff
@@ -110,17 +108,15 @@ def save_dic_list(self, filename, new_list):
         # Load current data
         current_list = []
         try:
-            # pylint: disable=invalid-name
-            f = open(filename, 'r')
-            current_list = json.load(f)
-            f.close()
+            fil = open(filename, 'r')
```

Comment:
maybe better if you name the var like "new_file" or "{prefix with context}_file"